tests/functional/conftest.py

```diff
@@ -293,6 +323,7 @@ def project(gl):
     name = f"test-project-{_id}"
 
     project = gl.projects.create(name=name)
+    project_id = project.id
```

Comment:
Are these changes to do with pass-by-assignment? Asking because a lot of our docs and tests have been happily passing `resource.id` directly to methods for a while it seems with no issue.